pages/dkp/kommander/1.3/release-notes/index.md

```diff
@@ -19,6 +19,13 @@ To get started with Kommander, [download](/dkp/konvoy/latest/download/) and [ins
 
 <p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
 
+# Important changes
```

Comment:
@russurquhart1  can we put this in 1.2 as well?

Reply:
Here is the JIRA for this:

https://jira.d2iq.com/browse/D2IQ-73609

Reply:
Put in 1.2 Release Notes.